supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -128,4 +128,30 @@ Avoid long introductions and unnecessary context. Shorten unnecessarily long sen
 === Principle 5: Error recovery, verification, and troubleshooting
 Recognize that people make mistakes and need to verify that they have completed a task. Be sure to include troubleshooting, error recovery, and verification steps.
 
+[[users]]
+== Users
+In most cases, the word "user" refers to a person or a person's user account, and therefore would be considered animate. In these cases, use animate personal pronouns such as "who".
+
+However, in certain technical cases, these users are not persons but instead user accounts or more abstract concepts. For example, Linux users such as root and guest do not relate to any person. Applications and services might run as specific Linux users with no person controlling them. SELinux users such as `user_u` or `sysadm_u` are identifiers of one or multiple Linux users for access control purposes.
+
+In these specific cases, and only if you cannot write around it, you can refer to these inanimate users with inanimate personal pronouns such as "that".
+
+.Examples
+* Animate user
++
+image:images/no.png[no] Experienced _users that_ can configure their own systems...
++
+image:images/yes.png[yes] _Users who_ want to install their own packages...
+
+* Inanimate user
++
+image:images/no.png[no] A Linux user has the restrictions of the _SELinux user who_ it is assigned to.
++
+image:images/no.png[no] A Linux user has the restrictions of the _SELinux user_ to _whom_ it is assigned.
++
+image:images/yes.png[yes] Specify a _user that_ is allowed to perform the requested action.
++
+image:images/yes.png[yes] A Linux user has the restrictions of the _SELinux user that_ it is assigned to.
```

Comment:
`... it is assigned to.`  Here `it` is inanimate and I believe it is referring to `A Linux user`. But IIUC, `Linux user` is still animate here, in which case we might have to say ..`they are assigned to.` instead. Feel free to correct me here.

Reply:
No, the Linux user here does not refer to a person who uses linux, but to the Linux user account, like `guest`.

Reply:
so we may want to emphasize on that too since it is overlooked currently.
e.g.
image:images/no.png[no] A _Linux user_ has the restrictions of the _SELinux user who it is_ assigned to.

Reply:
Hm, I tried to emphasize only the parts of speech to which the pronouns point (and the pronouns themselves). I think emphasizing this user would make it less clear, because no pronoun refers to it.

Reply:
> `... it is assigned to.` Here `it` is inanimate and I believe it is referring to `A Linux user`. But IIUC, `Linux user` is still animate here, in which case we might have to say ..`they are assigned to.` instead. Feel free to correct me here.

I thought `it` refers to `Linux user`